www/scripts/codecheckerviewer/BugViewer.js

```diff
@@ -433,14 +433,15 @@ function (declare, domClass, dom, style, fx, Toggler, on, query, Memory,
         that.bugStore.put(item);
       });
 
-      var filepath = getProperFilePath(this.reportData.checkedFile);
+      var filter_file = new CC_OBJECTS.ReportFilter();
```

Comment:
We use camel case variable names in JavaScript. Please rename it to `fileFilter`.